app/models/news_item.rb

```diff
@@ -1,7 +1,13 @@
 class NewsItem < ActiveRecord::Base
   extend Enumerize
+  belongs_to :published_by, foreign_key: :user_id, class_name: 'User'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.